zephyr/Kconfig

```diff
@@ -0,0 +1,680 @@
+# Copyright (c) 2017-2020 Linaro Limited
+# Copyright (c) 2020 Arm Limited
+# Copyright (c) 2022 Legrand North America, LLC.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+
+# Option to build the project with the MCUBoot application
+config MCUBOOT
+	default n
+	bool  "MCUBoot interfaces"
```

Comment:
Why is this selectable?

Reply:
Good question.  Looking back from an evolved PR, this probably shouldn't be.  I'll rework.

Reply:
In retrospective, this probably shouldn't be.  I'll fix it shortly.